Library/Formula/slacknimate.rb

```diff
@@ -0,0 +1,24 @@
+class Slacknimate < Formula
+  desc "text animation for Slack messages"
+  homepage "https://github.com/mroth/slacknimate"
+  url "https://github.com/mroth/slacknimate/archive/v1.0.0.tar.gz"
+  sha256 "7d495e2233a379f3ffffb2445f4b41b4306e52049d71f4c4c873786bb4048f45"
+  head "https://github.com/mroth/slacknimate.git"
+
+  depends_on "go" => :build
+  depends_on "glide" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    mkdir_p buildpath/"src/github.com/mroth"
+    ln_s buildpath, buildpath/"src/github.com/mroth/slacknimate"
+    system "glide", "install"
+    system "go", "build", "-o", "bin/slacknimate", "github.com/mroth/slacknimate"
+    bin.install "bin/slacknimate"
+  end
+
+  test do
+    system "#{bin}/slacknimate", "--version"
+    system "#{bin}/slacknimate", "--help"
```

Comment:
Can’t we write a test that does something more substantial than `--version`/`--help`?